src/components/chat-components/ChatInput.tsx

```diff
@@ -86,52 +90,19 @@ const ChatInput = forwardRef<{ focus: () => void }, ChatInputProps>(
       },
     }));
 
-    const debounce = <T extends (...args: any[]) => any>(
```

Comment:
the debounce function and the effect that listens to input message change do the same thing. It caused duplicated context values. Tested that everything still works after removing all the debounce calls